pyuvdata/uvdata/uvdata.py

```diff
@@ -1122,20 +1198,30 @@ def _add_phase_center(
         if not isinstance(cat_name, str):
             raise ValueError("cat_name must be a string.")
 
-        # The catalog name "unphased" is used internally whenever we have to make a
-        # block of data as unphased in a data set. To avoid naming collisions, check
+        if cat_type == "unphased":
+            warnings.warn(
+                "The 'unphased' catalog type has been renamed to 'unprojected'. Using "
+                "unprojected for now, this warning will become an error in version 2.4",
+                DeprecationWarning,
+            )
+            cat_type = "unprojected"
+            if cat_name == "unphased":
+                cat_name = "unprojected"
+
+        # The catalog name "unprojected" is used internally whenever we have to make a
+        # block of data as unprojected in a data set. To avoid naming collisions, check
         # that someone hasn't tried to use it for any other purpose.
-        if (cat_name == "unphased") and (cat_type != "unphased"):
+        if (cat_name == "unprojected") and (cat_type != "unprojected"):
             raise ValueError(
-                "The name unphased is reserved. Please choose another value for "
+                "The name unprojected is reserved. Please choose another value for "
                 "cat_name."
             )
 
         # We currently only have 4 supported types -- make sure the user supplied
         # one of those
-        if cat_type not in ["sidereal", "ephem", "driftscan", "unphased"]:
+        if cat_type not in ["sidereal", "ephem", "driftscan", "unprojected"]:
```

Comment:
In general I like enums. I don't know much about python enums (they didn't use to exist). Reading about them now...

Reply:
I looked into this. I think it might be nice, but it is going to be a lot of changes and we'd still have to do error handling to provide useful error messages. I think we should explore it more and if we do it, it should be in another PR.